dll/cpl/sysdm/lang/ro-RO.rc

```diff
@@ -34,236 +34,236 @@ END
-    AUTOCHECKBOX "T&ransmite alertă administrativă", IDC_STRRECSENDALERT, 14, 138, 148, 10
-    AUTOCHECKBOX "Repornește &automat sistemul", IDC_STRRECRESTART, 14, 152, 145, 10
-    GROUPBOX "Scriere informații de depanare", IDC_STATIC, 12, 167, 227, 76
+    LTEXT "Pentru a edita fişierul cu opţiuni de pornire, faceţi clic pe Editare.", IDC_STATIC, 14, 89, 187, 8
+    PUSHBUTTON "&Editare", IDC_STRRECEDIT, 188, 87, 50, 14
+    GROUPBOX "Pană de sistem", IDC_STATIC, 7, 111, 238, 140
+    AUTOCHECKBOX "Se &scrie un eveniment în jurnalul de sistem", IDC_STRRECWRITEEVENT, 14, 124, 178, 10
+    AUTOCHECKBOX "Se tri&mite o alertă administrativă", IDC_STRRECSENDALERT, 14, 138, 148, 10
+    AUTOCHECKBOX "&Repornire automată", IDC_STRRECRESTART, 14, 152, 145, 10
+    GROUPBOX "Scriere informaţii de depanare", IDC_STATIC, 12, 167, 227, 76
     COMBOBOX IDC_STRRECDEBUGCOMBO, 17, 180, 151, 49, CBS_DROPDOWNLIST | WS_TABSTOP
-    LTEXT "Fișierul de &depanare („cutie neagră”):", IDC_STATIC, 17, 198, 190, 8
+    LTEXT "&Fişier imagine memorie:", IDC_STATIC, 17, 198, 190, 8
     EDITTEXT IDC_STRRECDUMPFILE, 17, 210, 153, 12, ES_AUTOHSCROLL
-    AUTOCHECKBOX "S&uprascrie fișierul de depanare", IDC_STRRECOVERWRITE, 17, 228, 155, 10
-    DEFPUSHBUTTON "Con&firmă", IDOK, 141, 258, 50, 15
-    PUSHBUTTON "A&nulează", IDCANCEL, 195, 258, 50, 15
+    AUTOCHECKBOX "Se s&uprascrie orice fişier existent", IDC_STRRECOVERWRITE, 17, 228, 155, 10
+    DEFPUSHBUTTON "OK", IDOK, 141, 258, 50, 15
+    PUSHBUTTON "Revocare", IDCANCEL, 195, 258, 50, 15
 END
 
 IDD_VIRTMEM DIALOGEX 6, 6, 223, 248
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
 CAPTION "Memorie virtuală"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Unitate [etichetă volum]", IDC_STATIC, 12, 5, 96, 9
-    LTEXT "Limite de paginare (Mo)", IDC_STATIC, 135, 5, 96, 9
+    LTEXT "&Unitate [Etichetă volum]", IDC_STATIC, 12, 5, 96, 9
+    LTEXT "Dimensiune fişier de paginare (MO)", IDC_STATIC, 135, 5, 96, 9
     LISTBOX IDC_PAGEFILELIST, 10, 16, 204, 52, LBS_NOTIFY | LBS_USETABSTOPS
-    GROUPBOX "Dimensiune fișier de paginare (din selecție)", IDC_DRIVEGROUP, 10, 70, 204, 104
+    GROUPBOX "Dimensiune fişier de paginare pentru unitatea selectată", IDC_DRIVEGROUP, 10, 70, 204, 104
     LTEXT "Unitate:", IDC_STATIC, 20, 80, 50, 8
     LTEXT "%s", IDC_DRIVE, 88, 80, 100, 9
-    LTEXT "Spațiu disponibil:", IDC_STATIC, 20, 92, 56, 9
+    LTEXT "Spaţiu disponibil:", IDC_STATIC, 20, 92, 56, 9
     LTEXT "%s", IDC_SPACEAVAIL, 88, 92, 104, 9
-    LTEXT "Di&mensiunea inițială (Mo):", -1, 32, 118, 87, 9
-    LTEXT "Dimensiunea ma&ximă (Mo):", -1, 32, 131, 87, 9
-    AUTORADIOBUTTON "Limite parti&cularizate:", IDC_CUSTOM, 20, 105, 154, 9, WS_GROUP
-    AUTORADIOBUTTON "&Limite gestionate de sistem", IDC_SYSMANSIZE, 20, 145, 188, 9
-    AUTORADIOBUTTON "Fă&ră paginare", IDC_NOPAGEFILE, 20, 158, 78,9
+    LTEXT "D&imensiune iniţială (MO):", -1, 32, 118, 87, 9
```

Comment:
Ms-ro-RO-pic proving the &i accelerator please.

Reply:
![2024-05-05 17_58_50-Resource Hacker - sysdm cpl mui](https://github.com/reactos/reactos/assets/113553683/e48c7f76-50d3-4909-9815-1b635ad7ae7e)

Reply:
Thanks. MS uses 118 width. We do only use 87. Can you attach one more pic of the ros dlg, that proves the 87 is not causing truncation yet? Thx in advance.

Reply:
Resolved. Doesn't matter actually whether the 87 would overflow, because you didn't increase the strings length.